auth-api/src/auth_api/schemas/base_schema.py

```diff
@@ -18,43 +18,40 @@
 from auth_api.models import ma
 
 
-class BaseSchema(ma.ModelSchema):  # pylint: disable=too-many-ancestors
+class BaseSchema(ma.SQLAlchemyAutoSchema):  # pylint: disable=too-many-ancestors
     """Base Schema."""
 
     def __init__(self, *args, **kwargs):
         """Excludes versions. Otherwise database will query <name>_versions table."""
-        if hasattr(self.opts.model, 'versions') and (len(self.opts.fields) == 0):
-            self.opts.exclude += ('versions',)
+        if hasattr(self.opts.model, "versions") and (len(self.opts.fields) == 0):
```

Comment:
I don't think you need this anymore? Because we don't have a relationship for versions?